core/components/atoms/dropdown/__stories__/index.story.jsx

```diff
@@ -98,7 +99,7 @@ export const all = () => {
   const key = `checkbox${withCheckbox} applyButton:${showApplyButton} search:${withSearch}`;
 
   return (
-    <div style={{ width: '170px', marginLeft: '128px' }} key={key}>
+    <div key={key} className="Dropdown-alignment">
```

Comment:
Please remove this classname and create a similar story for both canvas and docs